content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+---
+title: Connections
+meta_description: "Establish and maintain a persistent connection to Ably using the Realtime Client Library SDK."
+section: realtime
+index: 20
+languages:
+  - javascript
+  - nodejs
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - flutter
+jump_to:
+  Help with:
+    - Introduction#intro
+    - Getting started#title
+    - Connection state explained
+    - Handling failures
+    - Connection state recovery
+    - Heartbeats#heartbeats
+    - API Reference#api-reference
+
+
+what are connections, mention states exist
+(repeating box) realtime only
+Create a connection
+(caveat box) connection is implicit with instantiation but this section cover explicit call to connect + usecase for when to use this option
+connect method + code example
+Monitor connections to an app
+From FAQ
+Connection metachannel and limitations of it. This will also cover dev console.
+Connection IDs
+(caveat box) Connections across multiple browser tabs
+Heartbeats
+explanation
+Code example + ping
+Close a connection
+explanation, incl 2min delay if not explicitly closed = peak rate + code example
+---
+
+h2(#intro). Introduction
+
+p(note). This section applies to the Ably Realtime Client only.
+
+The Ably Realtime library establishes and maintains a connection to the Ably service, using the most efficient transport available, typically "WebSockets":https://ably.com/topic/websockets. The Ably realtime protocol operates and multiplexes all channel traffic over that connection. Channels are separate from Connections and are used to logically separate messages into different topics, see the "channels documentation":/realtime/channels for further information.
+
+h3(#getting-started). Getting started: creating a connection
+
+p(caveat box). Connection is implicit with instantiation but this section covers explicit calls to connect and use-cases for when to use this option.
```

Comment:
Should we be using CAVEAT-BOX as per Miro? I also wasn't quite clear on what this sentence meant - could we clarify somehow? Not sure why use cases is hyphenated.

Reply:
Done